portfolio/src/main/webapp/sports.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sports</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div class="sportsheader">
+        <h1>Sports</h1>
+      </div>
+    </header>
+    <nav>
+      <div class="navbar">
+        <a href="index.html">Home</a>
+        <a href="dogs.html">Dogs</a>
+        <a href="sports.html">Sports</a>
+        <a href="baking.html">Baking</a>
+      </div>
+    </nav>
+    <div class="sportsrow">
+      <div class="sportscolumn" style="background-color: #ffff66;">
+        <h2>Soccer</h2>
+          <p>
+            I started playing soccer when I was only 5 years old. My parents were forcing me to try every sport possible, but soccer was what really stuck.
```

Comment:
What others you tries? Perhaps badminton? LOL

Reply:
My parents forced me to try just about everything but badminton wasn't one of them lol. I've played softball, tennis, and basketball other than the ones listed. I used to play a lot of backyard football too.